cmd/minikube/cmd/start_flags.go

```diff
@@ -285,6 +286,9 @@ func initDriverFlags() {
 
 	// qemu
 	startCmd.Flags().String(qemuFirmwarePath, "", "Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share")
+
+	// vfkit
+	startCmd.Flags().Bool(rosetta, false, "Enable Rosetta to support apps built for Intel processor on a Mac with Apple silicon (vfkit driver only)")
```

Comment:
does it also support running images built for x86 on all container runtimes ?
or is it only for binaries ? if also images, should mention in the help text.

it would be nice to check or verify (
suggestion, add an integration test that pulls an x86 image and runs it on silicon)
and test be skipped on anything but macos arm64
(could be a subset of functional test)

if dont wanna add ftest, can also just add manual verification and add to the PR description.